package.json

```diff
@@ -950,25 +950,68 @@
                                 }
                             },
                             "platform": {
-                                "anyOf": [
+                                "oneOf": [
                                     {
-                                        "type": "string"
+                                        "type": "object",
+                                        "properties": {
+                                            "os": {
+                                                "anyOf": [
+                                                    {
+                                                        "type": "string",
```

Comment:
I don't think this approach is the one we want. Adding the platform to scaffolding will make a project no longer portable from a Windows machine to Linux or vice versa. Instead, we should at runtime either use the exact value they specify, or if one does not exist, try to infer one based on other clues.

Reply:
Correct me if my wrong but I can't seem to think of a scenario where the scaffolded `platform` would not work when going from a windows -> linux machine if the user selects linux during scaffolding steps

This wouldn't work for a windows container but that's the expected behavior.

Reply:
You're right. I think my brain is smooth yesterday. :smile:

Reply:
:sob::sob: